forward.c

```diff
@@ -25,13 +25,12 @@ substdio ssout = SUBSTDIO_FDBUF(mywrite,-1,outbuf,sizeof outbuf);
 
 char num[FMT_ULONG];
 
-void main(argc,argv)
-int argc;
-char **argv;
+int main(int argc, char **argv)
 {
   char *sender;
   char *dtline;
   char *qqx;
+  int i;
```

Comment:
Given `i` is also declared in `for (int i; ...; ...) { ... }`, is it really needed here?
Sorry if I missed it.